apps/web-giddh/src/app/settings/integration/setting.integration.component.ts

```diff
@@ -465,13 +471,15 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
     }
 
     private getGoogleCredentials() {
-        if (PRODUCTION_ENV || isElectron) {
+        if (PRODUCTION_ENV) {
             return {
-                GOOGLE_CLIENT_ID: '641015054140-3cl9c3kh18vctdjlrt9c8v0vs85dorv2.apps.googleusercontent.com'
+                GOOGLE_CLIENT_ID: '641015054140-3cl9c3kh18vctdjlrt9c8v0vs85dorv2.apps.googleusercontent.com',
```

Comment:
Make this dynamic from environment

Reply:
@ravinderwalkover  I will make it use from general service. but for that, we need to do that in sperate code clean up task,